.changeset/honest-hornets-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@atproto/lexicon": patch
+---
+
+Add the ability to instantiate a Lexicon from an iterable, and to use a Lexicon as iterable.
```

Comment:
This landed in https://github.com/bluesky-social/atproto/pull/2482 but didn't get a changeset/publish, so going to preserve this.

Reply:
This was left over from a rework of the git branches.  Can be removed indeed!